app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragment.kt

```diff
@@ -10,7 +10,7 @@ import org.oppia.app.topic.PROFILE_ID_ARGUMENT_KEY
 import org.oppia.app.topic.TOPIC_ID_ARGUMENT_KEY
 import javax.inject.Inject
 
-private const val KEY_SKILL_ID_LIST = "SKILL_ID_LIST"
+private const val KEY_SUBTOPIC_ID_LIST = "SKILL_ID_LIST"
```

Comment:
Should we change the value too?

Reply:
Better, I moved it to companion object:
`internal const val SUBTOPIC_ID_LIST_ARGUMENT_KEY = "SUBTOPIC_ID_LIST_ARGUMENT_KEY"`

Reply:
Thanks